### PR TITLE
Deleteaccount

### PR DESCRIPTION
**Pull request by ijobling**
add delete account and from Penn feedback example, downloading student workspace

### CODE REVIEW

File: source/common/settings/user-profile.rst [hunk 1]
```diff
@@ -7,4 +7,15 @@ User Profile
...
+
+Delete your account
+-------------------
+
+You can delete your account at any time from the **Codio > Account** menu item and going to the **Account** tab
+
+   .. image:: /img/delete.png
+      :alt: Delete account
+      
+
```
Comment (shajason) [inline source/common/settings/user-profile.rst:20]: **Warning:** If you delete your account it cannot be retrieved nor can its contents be restored. To archive your data for potential later usage, export your projects or push them to a remote depository.

File: source/instructors/teaching/viewstudentwork.rst [hunk 2]
```diff
@@ -29,3 +29,11 @@ As a teacher you have access to your students cloud area within Codio. This allo
 
   .. image:: /img/openstudent.png
      :alt: Open student work
+
+Downloading students workspaces
```
Comment (shajason) [inline source/instructors/teaching/viewstudentwork.rst:33]: student's

File: source/instructors/teaching/viewstudentwork.rst [hunk 3]
```diff
@@ -29,3 +29,11 @@ As a teacher you have access to your students cloud area within Codio. This allo
 
   .. image:: /img/openstudent.png
      :alt: Open student work
```
Comment (shajason) [inline source/instructors/teaching/viewstudentwork.rst:31]: student

File: source/instructors/teaching/viewstudentwork.rst [hunk 4]
```diff
@@ -29,3 +29,11 @@ As a teacher you have access to your students cloud area within Codio. This allo
...
+
+Downloading students workspaces
+-------------------------------
+
+If you wish to download the students workspace you can do so either for individual students or for all students in the course:
```
Comment (shajason) [inline source/instructors/teaching/viewstudentwork.rst:36]: students'

**Review by ijobling — COMMENTED** [review]
done and repushed all ( I think )

**Review by shajason — APPROVED** [review]

**Pull request merged**